cloud/shared/bin/lib/terraform.py

```diff
@@ -12,6 +12,18 @@
 from cloud.aws.templates.aws_oidc.bin.aws_cli import AwsCli
 
 
+def find_variable_default(config: ConfigLoader,
```

Comment:
**suggestion:** Could we add a doc comment above this? This finds what the default value is set to for a variable or what the value is set to in the config during the run?